boards/arm/96b_carbon/Kconfig.defconfig

```diff
@@ -10,4 +10,47 @@ if BOARD_96B_CARBON
 config BOARD
 	default 96b_carbon
 
+if I2C
+
+config I2C_1
```

Comment:
One day I'd like to kill of the Kconfig.defconfig files, so anywhere we have a def_bool we really should just be doing a 'select' in the Kconfig.